configure_cmd.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"fmt"
+	"github.com/briandowns/spinner"
+	"github.com/urfave/cli/v2"
+	"os"
+	"os/exec"
+	"time"
+)
+
+// beforeSatelliteAction is run before satelliteAction is run. It is used
+// for checking CLI options.
+func beforeSatelliteAction(ctx *cli.Context) error {
+	// First check if machine-readable format is used
+	err := setupFormatOption(ctx)
+	if err != nil {
+		return err
+	}
+
+	satelliteUrlStr := ctx.String("url")
+	if satelliteUrlStr == "" {
```

Comment:
We can mark the `cli.Flag` defining this flag as `Required: true`. That makes this check unnecessary.